integration-cli/docker_cli_wait_test.go

```diff
@@ -120,3 +120,38 @@ func TestWaitBlockedExitRandom(t *testing.T) {
 
 	logDone("wait - blocking wait with random exit code")
 }
+
+func TestWaitKilledContainerNoWait(t *testing.T) {
+	defer deleteAllContainers()
+
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "/bin/cat")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+	containerID := strings.TrimSpace(out)
+
+	killCmd := exec.Command(dockerBinary, "kill", containerID)
+	out, _, err = runCommandWithOutput(killCmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+
+	errchan := make(chan error)
+	go func() {
+		waitCmd := exec.Command(dockerBinary, "wait", containerID)
+		_, _, err := runCommandWithOutput(waitCmd)
```

Comment:
Lol, now I wonder why we need runCommand*WithOutput* here :) maybe to chan we can pass:
```
fmt.Errorf("%v:\n%s", err, out)
```